src/main/java/de/tum/in/www1/artemis/repository/StudentExamRepository.java

```diff
@@ -10,4 +14,7 @@
  */
 @Repository
 public interface StudentExamRepository extends JpaRepository<StudentExam, Long> {
+
+    @Query("SELECT s FROM StudentExam s WHERE s.exam.id = :#{#examId}")
```

Comment:
We don't need the @query here, as the method name should suffice.
See https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#repositories.query-methods